roles/dropwizard-api/files/jenkins-run.j2

```diff
@@ -0,0 +1 @@
+exec sudo "{{ script_path }}" "$1"
```

Comment:
Wrapper script so that we don't have to modify our jenkins jobs to include "sudo" in the exec command field. Since we aren't going to run docker in api-dev anytime soon, this type of wrapper is necessary.